wp-content/themes/core/components/blocks/spacer/css/spacer.pcss

```diff
@@ -0,0 +1,40 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Component: Block: Spacer
+ *
+ * ----------------------------------------------------------------------------- */
+
+.b-spacer {
+	height: 24px;
+
+	@media (--viewport-medium) {
+		height: 48px;
+	}
+
+	/* CASE: Large Size */
+	&.b-spacer--large {
+		height: 48px;
+
+		@media (--viewport-medium) {
+			height: 96px;
+		}
```

Comment:
Can we use existing spacer vars rather than arbitrary px values?

Reply:
My concern was that design would update the spacer in the kitchen sink and effect this spacer block. If that's expected than thats fine but design seemed pretty specific on this set of pixels

Reply:
I think that would be an intentional change then, yes. It would affect any spacing throughout the system, including the spacing here/between blocks.